vendor/src/github.com/microsoft/hcsshim/createprocess.go

```diff
@@ -12,30 +14,71 @@ import (
 // processParameters is use to both the input of CreateProcessInComputeSystem
 // and to convert the parameters to JSON for passing onto the HCS
 type CreateProcessParams struct {
-	ApplicationName                   string
-	CommandLine                       string
-	WorkingDirectory                  string
-	StdInPipe, StdOutPipe, StdErrPipe string
```

Comment:
Out of curiosity, why not replace this line with 3 bools to prevent passing too many arguments to CreateProcessInComputeSystem ?